src/frontend/src/optimizer/plan_node/mod.rs

```diff
@@ -129,12 +128,6 @@ impl<T: PlanNode> From<T> for PlanRef {
     }
 }
 
-impl Display for PlanRef {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        std::fmt::Display::fmt(&self.0, f)
-    }
-}
```

Comment:
we can implement it with `plan.explain_to_string()`?

Reply:
Yes we can, but we don't have to. Currently there is no use case